students/tri_nguyen/lesson04/dict_lab.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  21-Feb-2018
+# ------------------------------------------- #
+
+import os
+
+# Declare functions
+
+
+def dict_one():
+    ''' practice Python dictionary '''
+
+    a_dict = {'name': 'Tri', 'city': 'Seattle', 'cake': 'cheese'}
+
+    # Display the dictionary
+
+    print(a_dict, '\n')
+
+    # Delete the entry for cake
+
+    del a_dict['cake']
+
+    print(a_dict, '\n')
+
+    # Add an entry for fruit with mango
+
+    a_dict['fruit'] = 'Mango'
+
+    print(a_dict, '\n')
+
+    # Display the dictionary keys
+    print('Keys of "a_dict" dictionary are:')
+
+    for key in a_dict:
+        print('+', key)
+
+    # Display the dictionary values
+    print('\nValues of "a_dict" dictionary are:')
+
+    for value in a_dict.values():
+        print('+', value)
+
+    # Check if "cake" is a key
+
+    if 'cake' in a_dict:
+        print('\nYes, there is cake.')
+    else:
+        print('\nNo, there is no cake')
+
+    # Check if "Mango" is a value
+
+    value_list = list(a_dict.values())
+
+    if 'Mango' in value_list:
```

Comment:
`dict.values()` returns a list of values so you can just loop through that without the `value_list` assignment: `if 'Mango' in a_dict.values():`